app/src/main/res/layout/list_comments_mini_item.xml

```diff
@@ -66,4 +66,24 @@
         android:textSize="@dimen/video_item_search_upload_date_text_size"
         tools:text="1 year ago" />
 
+    <Button
+        android:id="@+id/itemContentReplyButton"
+        style="?android:attr/borderlessButtonStyle"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:layout_below="@id/detail_thumbs_up_img_view"
+        android:layout_marginTop="4dp"
+        android:layout_toEndOf="@+id/itemThumbnailView"
+        android:background="?attr/selectableItemBackground"
+        android:gravity="start|center_vertical"
+        android:layout="@id/detail_heart_image_view"
+        android:minHeight="0dp"
+        android:paddingHorizontal="4dp"
+        android:paddingVertical="6dp"
+        android:textAppearance="?android:attr/textAppearanceLarge"
+        android:textColor="@color/show_replies_button_text_color"
+        android:textSize="@dimen/file_picker_items_text_size"
+        tools:text="0 Reply" />
```

Comment:
Not important, but wouldn't it be clearer if this was "3 replies"? "0 replies" is never really shown.